torch/fx/graph.py

```diff
@@ -922,11 +927,25 @@ def emit_node(node : Node):
             # have been emitted. To continue to have valid Python code, emit a
             # single pass statement
             body.append('pass\n')
+        if self._in_spec is not None:
+            assert(self._orig_args is not None)
```

Comment:
`_in_spec` and `_orig_args` should probably go together in a struct (e.g. `NamedTuple`) attribute if they're always paired together